launchable/commands/record/tests.py

```diff
@@ -80,19 +81,39 @@
 @click.pass_context
 def tests(context, base_path: str, session: Optional[str], build_name: Optional[str], post_chunk: int, subsetting_id: str,
           flavor, no_base_path_inference):
+    org, workspace = get_org_workspace()
+
+    if org is None or workspace is None:
+        raise click.UsageError(click.style(
+            "Could not identify Launchable organization/workspace. "
+            "Please confirm if you set LAUNCHABLE_TOKEN or LAUNCHABLE_ORGANIZATION and LAUNCHABLE_WORKSPACE environment variables",
+            fg="red"))
+
+    client = LaunchableClient(test_runner=context.invoked_subcommand,
```

Comment:
initialize client at the top and reuse it through the function

Reply:
I believe that the shorter the scope of the variable and the time it is used, the better, if possible. What is the reason for using them around?

Reply:
used only within a method is short enough, and I don't think taking overhead to initialize a client so often is good choice.